presidio-analyzer/tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+import pytest
+
+from presidio_analyzer.nlp_engine import NLP_ENGINES
+from presidio_analyzer.predefined_recognizers import NLP_RECOGNIZERS
+from presidio_analyzer.entity_recognizer import EntityRecognizer
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--runfast", action="store_true", default=False, help="run fast tests"
+    )
+
+
+def pytest_configure(config):
+    config.addinivalue_line("markers", "slow: mark test as slow to run")
```

Comment:
I think that I need to change the documentation here.  The default is to run the full model.  I changed the flag to `--runfast` so that it uses the small model and skips one test if you add that flag, but if there is no flag then it uses the big model and runs everything.

Reply:
I see now